docs/user/discover.asciidoc

```diff
@@ -3,114 +3,227 @@
+Most users have one of two goals in mind when exploring their data:
 
-[float]
-=== Search your data
+- **Get an overview of what is happening.**
+For example, you might want to look at the logs captured in your {es} cluster to
+get information on the health and performance of your cloud-deployed software over time.
+You can save your findings for later use when making decisions about maintenance and improvements.
+
+- **Find an answer to a specific question.** Consider if you are looking at
+Github statistics and are interested in the number of pull requests that went
+into a patch release and who contributed to the release. You want to visualize
+this data and present it to your coworkers.
+
+What is your goal?  The answer to this question will lead your explorations.
 
-Now that you have your data and set the time span, you can start asking your questions.
-You can search your data using the <<kuery-query, Kibana Query language>>,
-which offers a simplified query syntax.
-For example, if
-you search for `day_of_week : Friday`, you'll get a list of all documents
-in which `day_of_week` is set to `Friday`. If you prefer
-<<lucene-query, Lucene query syntax>>, you can access it from the KQL menu.
 
 [float]
-=== Filter your search results
+[[find-the-data-you-want-to-use]]
+=== Step 1. Find the data you want to use
 
-Next, you'll want narrow your search results to a more manageable data set.
-When you click on a name in the field list, you'll see
-the top five values for the field, the number of documents that contain the field,
-and the percentage of documents that contain each value. From this view, you can
-use the (+) magnifier icon to quickly find all
-documents that have that value, or (-) to exclude all
-documents with that value. For more filter options, see <<field-filter, filtering by field>>.
+Tell {kib} where to find your data, then set a time range for your exploration.
 
+. Open the main menu, and select **Discover**.
++
+With **Discover**, you can perform ad-hoc analysis of your data in real-time.
+It’s where you’ll find hidden insights and relationships and emerging trends in your data.
+
+. Select the data you want to work with by choosing an <<index-patterns,index pattern>>.
++
+{kib} requires an index pattern to tell it which Elasticsearch indices you want to work with.
+To view the ecommerce sample data, make sure the index pattern is set to **kibana_sample_data_ecommerce**.
++
 [role="screenshot"]
-image::images/filter-field.png[height=317]
+image::images/discover-index-pattern.png[How to set the index pattern in Discover]
+
+. Adjust the time range to *Last 7 days*.
++
+NOTE: The range selection is based on the default time field in your data.
+If you are using the sample data, this value was set when you added the data.
+If you  are using your own data, and it does not have a time field, the range selection is not available.
 
+. Optional. To view the count of documents for a given time,
```

Comment:
Super nit: To make this a full sentence, how about:

To optionally view the (number?) of documents for a given time, click the drag the mouse over the histogram, and then click **Refresh**.